build.sbt

```diff
@@ -89,8 +89,17 @@ def altPublishSettings: Seq[Setting[_]] =
     }
   )
 
+val noPublish: Seq[Setting[_]] = List(
+  publish := {},
+  publishLocal := {},
+  publishArtifact in Compile := false,
```

Comment:
Can we actually remove the `publish*` and `publishArtifact` redefinitions if `skip in publish` is already being respected?